codefresh-release.yml

```diff
@@ -15,7 +15,7 @@ steps:
 
   extract_version:
     title: "Exporting package.json version"
-    image: codefresh/cli-build
+    image: codefresh/build-cli
```

Comment:
what is this image?

Reply:
`codefresh/build-cli` is the image we provide instead of `codefresh/cli-build` - because we do not have any Dockerfile for it and cannot review or change it.